src/app/components/Toolbar/Features/Settings/index.tsx

```diff
@@ -0,0 +1,17 @@
+import { useContext } from 'react'
+import { Box } from 'grommet/es6/components/Box'
+import { ResponsiveContext } from 'grommet/es6/contexts/ResponsiveContext'
+import { ThemeSelect } from 'app/components/ThemeSwitcher'
+import { LanguageSelect } from 'app/components/LanguageSelect'
+import { layerOverlayMinHeight } from '../layer'
+
+export const Settings = () => {
```

Comment:
I named the button SettingsButton for all tabs in modal window. We should rename somehow

Reply:
Any proposal?